### ISSUE TITLE
Generating an interview schedule

### ISSUE THREAD

**Issue by dirkschumacher** [R4EPI/surveys#4]
Is is something we want to do? Together with sample size calculation and the sampling in R, maybe we can also generate other supporting material for operations? Just an idea

**Comment by epiamsterdam** [comment]
interview schedule for???

**Comment by dirkschumacher** [comment]
I thought I had seen that in one of the surveys. A plan when and how to visit people (my interpretation)

**Comment by epiamsterdam** [comment]
yes. you had. but i think this is hard to standardise for each survey as it will depend on numerous factors (including rains, road conditions, security, car availability etc.)

**Comment by aspina7** [comment]
simple helper function where it takes sample sized required - then you put in how many interview teams you have, the time days available for survey and maybe the average number of houses visited. Then assigns number of houses required for each team.  
Would then also be nice to link this to random gps point generation function - so teams are assigned to points clustered together (minimising walking time) - have code for this from bangladesh but needs to be cleaned up, remind me to share.

**Comment by aspina7** [comment]
so is a bit messy but chucked my code from bangladesh on the [google drive](https://drive.google.com/open?id=1g6vRZhjPh8jxAC4lyvLEflD0g_bAjwzk) too. 

Its messy because certain teams only worked in certain areas (and wrote it all in a hurry too. 

But the shortest distance stuff is in there from line 181. It could be improved by having structured starting points spread equally accross the area study area. 
I just took random start points equal to num of teams, then a distance matrix for each of those points to each of the survey points (distance matrix), and then assigned the x number of points team by team that were closest (naturally the last team in line gets the most walking... which would happen to a lesser degree if you spread out the starting points more equally).

**Comment by epiamsterdam** [comment]
For me this is a "nice to have" but not essential. Again, in camps it might be interesting to have this automatic planning function. Allthough I know that often the closest distance doesnt correspond to th eeasiest way to get there (as experienced by survey teams recently in the slums of Port au Prince in Haiti). In most surveys we wouldnt be able to plan like this as we would require car and road transportation and security would not allow the automatically generated plans.
